### PR TITLE
crimson/common: minor cleanups to futures

### PR DESCRIPTION
**Pull request by cyx1231st**
## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by cyx1231st** [comment]
```
[ RUN      ] Queue.SpawnAsyncRequest
../src/test/rgw/test_rgw_dmclock_scheduler.cc:425: Failure
Value of: context.stopped()
  Actual: false
Expected: true
[  FAILED  ] Queue.SpawnAsyncRequest (1 ms)

The following tests FAILED:
	203 - unittest_rgw_dmclock_scheduler (Failed)
```

File: src/crimson/common/errorator.h [hunk 1]
```diff
@@ -964,16 +964,6 @@ struct errorator {
   public:
     using type = ::crimson::errorator<AllowedErrors...>::future<ValueT>;
 
-    template <class Func, class... Args>
```
Comment (rzarzynski) [inline src/crimson/common/errorator.h:970]: Typo in commit msg – `s/futuruze/futurize/`.

**Review by rzarzynski — COMMENTED** [review]
Just a typo in the commit msg. Otherwise LGTM!

**Review by athanatos — APPROVED** [review]

**Review by liu-chunmei — APPROVED** [review]

**Comment by athanatos** [comment]
Just waiting on the commit message fix.

**Review by xxhdx1985126 — APPROVED** [review]

File: src/crimson/common/errorator.h (hunk 1 above)
Comment (cyx1231st) [reply src/crimson/common/errorator.h:970]: fixed

**Review by rzarzynski — APPROVED** [review]

**Pull request merged**